ui/packages/catalog-ui-search/src/main/webapp/component/query-settings/query-settings.view.js

```diff
@@ -55,26 +53,50 @@ define([
         initialize: function () {
             this.model = this.model._cloneOf ? store.getQueryById(this.model._cloneOf) : this.model;
             this.listenTo(this.model, 'change:sortField change:sortOrder change:src change:federation', Common.safeCallback(this.onBeforeShow));
+            if(properties.hasExperimentalEnabled())
+            {
+                this.resultFormCollection = ResultForm.getResultCollection();
+                this.listenTo(this.resultFormCollection, 'change:added', this.handleFormUpdate)
+            }
+        },
+        handleFormUpdate: function(newForm) {
+            this.renderResultForms(this.resultFormCollection.filteredList)
         },
         onBeforeShow: function () {
             this.setupSortFieldDropdown();
+            this.setupSrcDropdown();
             this.turnOnEditing();
-
-            let resultTemplates = ResultForm.getResultTemplatesProperties();
+            if(properties.hasExperimentalEnabled())
+            {
+                this.renderResultForms(this.resultFormCollection.filteredList)
+            }
+        },
+        renderResultForms: function(resultTemplates){
+            if(resultTemplates == undefined)
+            {
+                resultTemplates = [];
+            }
+            resultTemplates.push({
+                label: 'All Fields',
+                value: 'All Fields',
```

Comment:
✏️ Looks like the existing [query-settings.view](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/query-settings/query-settings.view.js#L145) and [search-form.view](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js#L97) are still expecting the `value` of the `All Fields` enum option to be `allFields`. It doesn't entirely matter which, but I would say that we should normalize it